src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {

```

Comment:
Don't extend your `Main/Application` class from `FigureSupplier` or `ColorSupplier`. To be able to call the non-static method, we just need to create an instance of the class. It seems you've done this correctly, but keep in mind not to create multiple instances when one would suffice.